src/components/Messages/Message/index.js

```diff
@@ -67,6 +67,16 @@ const getSystemMessageText = ({ t, conversationFinishedMessage }) =>
 	|| (t === MESSAGE_TYPE_WELCOME && I18n.t('Welcome'))
 	|| (t === MESSAGE_TYPE_LIVECHAT_CLOSED && (conversationFinishedMessage || I18n.t('Conversation finished')));
 
+const getMessageUsernames = (compact, message) => {
+	if (compact) {
+		return [];
+	}
+	if (message.alias && message.u && message.u.name) {
```

Comment:
Why do check the `alias` property?

Reply:
because alias is there only for guest and not for agent

Reply:
Here alias is the possible way to distinguish between guest and agent

Reply:
No. If you check the server-side, the alias is available in other situations.
My suggestion is not to check it.

Reply:
Then how can we distinguish between the guest and agent since name and username properties are common to both. 
Should I check it with regex of `guest-xx` or there is some other way which I don't know? Please help

Reply:
Okay. I'll suggest another implementation.